development/grafana-setup/entrypoint.sh

```diff
@@ -12,6 +13,9 @@ done
 
 echo "Up!"
 
+echo "Adding database for InfluxDB"
+curl -i -XPOST ${INFLUXDB_API} --data-urlencode "q=CREATE DATABASE sitespeedio"
```

Comment:
I'm looking in [the documentation](https://hub.docker.com/_/influxdb/), and they use GET for creating a database, like this:
```
curl -G ${INFLUXDB_API} --data-urlencode "q=CREATE DATABASE sitespeedio"
```